llvm/lib/TargetParser/X86TargetParser.cpp

```diff
@@ -171,14 +171,14 @@ constexpr FeatureBitset FeaturesClearwaterforest =
 
 // Geode Processor.
 constexpr FeatureBitset FeaturesGeode =
-    FeatureX87 | FeatureCMPXCHG8B | FeatureMMX | Feature3DNOW | Feature3DNOWA;
```

Comment:
Do we need to add a FeaturePREFTECH of some kind?

Reply:
Possibly not?

We didn't enable it before (so, e.g. `__PRFCHW__` isn't and wasn't defined), and while we DID enable the prefetchw instruction in the backend for these targets, we still do now (via implying it in X86.td).

OTOH, maybe it's weird to have that divergence, so I'd also be fine with adding it (and therefore setting the preprocessor define).

Reply:
(I somewhat lean towards submitting the commit as-is, but will change this if you wish)

Reply:
I think we should add the define. If/when we eventually remove mm3dnow.h then the only way to access prfchwintrin.h is via x86intrin.h (which we already recommend) which can sometimes require the `__PRFCHW__` define:
https://github.com/llvm/llvm-project/blob/f6add66b720f85bf1092af7d6702b7397da57349/clang/lib/Headers/x86intrin.h#L17-L19

Also, we still have references to mm3dnow.h in prfchwintrin.h that probably need removing:
https://github.com/llvm/llvm-project/blob/f6add66b720f85bf1092af7d6702b7397da57349/clang/lib/Headers/prfchwintrin.h#L10-L12

Reply:
OK, making the change.

I note that "sometimes" is essentially "if `__SCE__` is defined". Really weird to have that be Sony-specific behavior.